### PR TITLE
Returns first

### PR DESCRIPTION
**Pull request by Aareksio**
Code cleanup. Return first to avoid unnecessary intends.

### CODE REVIEW

**Comment by DoctorMcKay** [comment]
I'm not sure I want to merge what's essentially a style choice, introducing the potential for unforeseen bugs and behavior changes.

**Comment by Aareksio** [comment]
Indeed it is just a small style change - the point is to make the module style consistent, and I see return first style in the most parts of the module.

I fully understand if you don't have time to review this or just don't agree.

**Comment by DoctorMcKay** [comment]
I'd be more inclined to merge this if someone reviewed it, but unfortunately at the moment I don't have the time.

File: lib/assets.js [hunk 1]
```diff
@@ -69,9 +69,10 @@ TradeOfferManager.prototype._addDescriptions = function(items, callback) {
 	this._requestDescriptions(descriptionRequired, (err) => {
 		if (err) {
 			callback(err);
-		} else {
-			callback(null, this._mapItemsToDescriptions(null, null, items));
+			return;
```
Comment (andrewda) [inline lib/assets.js]: You should probably either change this to `return callback(err);` or split line 152 into two separate lines like this.

Comment (DoctorMcKay) [inline lib/assets.js]: For what it's worth, while `return callback()` is more compact, I dislike it as it makes my IDE yell at me for only returning a value sometimes.

Comment (andrewda) [inline lib/assets.js]: I agree. Just want things to be consistent.

Comment (Aareksio) [inline lib/assets.js]: Yeah, I found it strange and after reviewing the code once again I found that L152 is the only place `return callback();` is used. 

I'll split that.

Comment (Aareksio) [inline lib/assets.js]: Done.

**Comment by ylms** [comment]
Will it actually faster the process?

**Comment by ylms** [comment]
Will it actually speed up the process?

**Comment by Aareksio** [comment]
@ylms Unnoticeable. It'll make the module code style more consistent though.

**Comment by DoctorMcKay** [comment]
Sorry, I think I'll have to pass on this.

**Pull request closed without merging**